### PR TITLE
Develop pos, updates of Nexo Retailer and dot net

### PR DESCRIPTION
**Pull request by the-anders-jarold**
Updates regards what the new terminal version has to offer. PrintRequest to terminal and CVM Signature even for client only mode.

### CODE REVIEW

File: pax-terminal/Nexo-Retailer/use-a920-printer.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+{:table .table.striped}
+| Name | Lev | Attribute | Description
+|:---|:--:|:--|:--|
+| PrintRequest | 1 | | |
+| PrintOutput | 2 | DocumentQualifier | Value: SaleReceipt |
+|  |  | ResponseMode | Value: PrintEnd |
+| OutputContent | 3 | OutputFormat | Value: Text |
+| OutputText | 4 | Text to be printed. | A new line is generated per OutputText. However, a text may span several rows if a  byte of hex 0A is used as line break |
+|  |  | StartRow | Vertical pixel paper feed. It is recommended to end with an empty OutputText with StartRow set to 224 to feed paper past the cutter. |
+|  |  | Alignment | Enumeration: Left, Centred, Right, Justified |
```
[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/Nexo-Retailer/use-a920-printer.md:42]: That's how it is spelled in Nexo.

File: pax-terminal/Nexo-Retailer/use-a920-printer.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+<SaleToPOIRequest>
+    <MessageHeader DeviceID="25" MessageCategory="Print" MessageClass="Device" MessageType="Request" POIID="A-POIID" SaleID="1" ServiceID="29"/>
+    <PrintRequest>
+        <PrintOutput DocumentQualifier="SaleReceipt" ResponseMode="PrintEnd">
+            <OutputContent OutputFormat="Text">
+                <OutputText Alignment="Left" CharacterHeight="HalfHeight" CharacterWidth="SingleWidth" Color="Black">8x16 font size</OutputText>
+                <OutputText Alignment="Left" CharacterWidth="SingleWidth" Color="Black">12x24 font size</OutputText>
+                <OutputText Alignment="Justified" CharacterHeight="SingleHeight" CharacterWidth="SingleWidth" Color="Black" StartRow="20">8x32   Justified alignment</OutputText>
+                <OutputText Alignment="Left" CharacterHeight="DoubleHeight" CharacterWidth="SingleWidth" Color="White" StartRow="40">12x48 black background</OutputText>
+                <OutputText Alignment="Centred" CharacterHeight="HalfHeight" CharacterWidth="DoubleWidth" Color="Black" StartRow="50">Centered 16x16</OutputText>
```
[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/Nexo-Retailer/use-a920-printer.md:62]: That's how it is spelled in Nexo.

File: pax-terminal/Nexo-Retailer/fuelfunctionality.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+| SaleTransactionID | 3 | TimeStamp| Request timestamp local time with offset from GMT. |
+|   | | TransactionID | ID of the transaction set by the POS. This value is sent to the PosPay server, but is unfortunately not forwarded to clearing.|
+| PaymentTransaction | 2 | | |
+| AmountsReq | 3 | RequestedAmount | The total amount of transaction as a decimal value. Use a **'.'** for decimal point if needed. |
+|   | | Currency | As a 3 letter abbrivation - ISO-4217. Available are `SEK`, `NOK`, `DKK`, `EUR`.|
+|  | | CashBackAmount | As `whereof`value. Is included in `RequestedAmount`. |
+| SaleItem | 3 | ItemId | 0-n Required. |
+|  | | ProductCode | DigitString 1-12 bytes. Required. |
+|  | | ItemAmount | Decimal: Total amount of the SaleItem. Use a **'.'** for decimal point. Required.|
+| UnitOfMeasure|4| |Only present together with Quantity. Enumeration: Litre or Kilogram. |
```
[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/Nexo-Retailer/fuelfunctionality.md:32]: That's how it is spelled in Nexo.

File: pax-terminal/Nexo-Retailer/fuelfunctionality.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+ <SaleToPOIRequest>
+    <MessageHeader MessageCategory="Payment" MessageClass="Service" MessageType="Request" POIID="A-POIID" ProtocolVersion="3.1" SaleID="1" ServiceID="1524253497"/>
+    <PaymentRequest>
+        <SaleData TokenRequestedType="Customer">
+            <SaleTransactionID TimeStamp="2023-09-08T16:17:32.9834651+02:00" TransactionID="1524253496"/>
+        </SaleData>
+        <PaymentTransaction>
+            <AmountsReq CashBackAmount="0" Currency="SEK" RequestedAmount="514.90"/>
+            <SaleItem ItemID="0" ItemAmount="446.90" ProductCode="24601">
+                <UnitOfMeasure>LITRE</UnitOfMeasure>
```
[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/Nexo-Retailer/fuelfunctionality.md:52]: That's how it is spelled in Nexo.

File: pax-terminal/NET/saleitem.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+description: |
+    SaleItem is used for fuel transactions. A list of sale items is included i TransactionSetup when calling Payment.
+---
+
+```c#
+namespace SwpTrmLib.Nexo 
+{
+    public class SaleItem
+    {
+        public enum UnitsOfMeasure { None, Litre, Centilitre, Kilometre, Kilogram, Gram, Metre, Centimetre, Other};
```
[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/NET/saleitem.md:12]: That's how it is spelled in Nexo.

[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/NET/saleitem.md:12]: That's how it is spelled in Nexo.

File: pax-terminal/NET/saleitem.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+        public XElement XML();
+        public override string ToString();
+    }
+}
+```
+
+{:.table .table-striped}
+| :------------- | :-------------- |:--------------- |:---|
+| **Type** | **Name** | **Description** | |
+| UnitOfMeasure | Enumeration UnitOfMeasures |Litre or Kilogram | Mandatory |
```
[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/NET/saleitem.md:36]: That's how it is spelled in Nexo.

File: pax-terminal/NET/saleitem.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/NET/saleitem.md:12]: That's how it is spelled in Nexo.

File: pax-terminal/Nexo-Retailer/use-a920-printer.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+                <OutputText Alignment="Centred" CharacterWidth="DoubleWidth" Color="Black" StartRow="20">Centered 24x24</OutputText>
```
[the remark this answers is not in the document]
Comment (the-anders-jarold) [reply pax-terminal/Nexo-Retailer/use-a920-printer.md:63]: That's how it is spelled in Nexo.

**Review by arebra — APPROVED** [review]

**Pull request merged**